HIP-Basic/CMakeLists.txt

```diff
@@ -25,8 +25,45 @@ project(HIP-Basic LANGUAGES CXX)
 
 # Only supported on HIP (not CUDA)
 if(NOT "${GPU_RUNTIME}" STREQUAL "CUDA")
-    add_subdirectory(assembly_to_executable)
-    add_subdirectory(llvm_ir_to_executable)
+
```

Comment:
```suggestion

    # ROCm installation path
    if(WIN32)
        set(ROCM_ROOT "$ENV{HIP_PATH}" CACHE PATH "Root directory of the ROCm installation")
    else()
        set(ROCM_ROOT "/opt/rocm" CACHE PATH "Root directory of the ROCm installation")
    endif()
    
```
ROCM_ROOT hasn't been defined yet, if you're running cmake directly from the "HIP-Basic" directory. For example, if the user just wants to see the HIP related examples.
In testing, I found that on the first run of `cmake -B build` none of the programs are found. However, subsequent runs would find them.

Reply:
Fixed. Thanks!